Lib/importlib/resources.py

```diff
@@ -267,11 +267,12 @@ def __init__(self, zipimporter, fullname):
         self.fullname = fullname
 
     def open_resource(self, resource):
-        path = f'{self.fullname}/{resource}'
+        fullname_as_path = self.fullname.replace('.', '/')
+        path = f'{fullname_as_path}/{resource}'
         try:
             return BytesIO(self.zipimporter.get_data(path))
         except OSError:
-            raise FileNotFoundError
+            raise FileNotFoundError(path)
```

Comment:
Maybe use the `repr()` to help identify potentially odd things like whitespace issues?

Reply:
I thought about that, but I wanted the actual path string as the argument to the exception.  I guess it's not *that* big a deal either way.